docs/how-to/llm-fine-tuning-optimization/llm-inference-frameworks.rst

```diff
@@ -132,7 +123,7 @@ Installing vLLM
             ROCR_VISIBLE_DEVICES=0,1 python -m vllm.entrypoints.api_server --model /data/llama-2-7b-chat-hf --dtype float16 –tp 2 --port 8000 &
             ROCR_VISIBLE_DEVICES=2,3 python -m vllm.entrypoints.api_server --model /data/llama-2-7b-chat-hf --dtype float16 –tp 2--port 8001 &
 
-      6. To test, send it a curl request containing a prompt.
+      5. To test, send it a curl request containing a prompt.
 
          .. code-block:: shell
 
```

Comment:
Also, below line 171

2. Launch a model using TGI server on a single accelerator.
->
2. Inside the container, launch a model using TGI server on a single accelerator.

Reply:
done